functions/src/api/openai.api.ts

```diff
@@ -0,0 +1,79 @@
+import OpenAI from "openai"
+
+const MAX_TOKENS_FINISH_REASON = "length";
+
+export async function callOpenAITextCompletion(
+  apiKey: string,
+  prompt: string,
+  modelName: string
+) {
+  console.log('callOpenAITextCompletion')
```

Comment:
Remove?

Reply:
You meant the console.log line, right? Done. (Also changed the one below to console.error)